ingest_data_Lynn.R

```diff
@@ -0,0 +1,236 @@
+# This script loads in the filtered data and checks for invalid rows, as well as changes the
+# neighborhood names to be compatible with the neighborhoods table in the data base.
+# It then inserts the data into the data base.
+
+# General setup-----
+
+library(RPostgreSQL)
+con <- dbConnect(PostgreSQL(), user="lhk", password = "0ntBNyCcuipdh8g", 
+                 dbname="lhk", host="pg.stat.cmu.edu")
+
+# Change the schema that I will access with the connection
+set.schema <- dbSendStatement(con, "set search_path to 'schemacrime';")
+
+library(testthat)
```

Comment:
Yes, but your testthat usage seems to be in your Rmd file, not in this file -- generally it's best to keep `library` calls in the files that use them, so it's obvious which code uses which libraries.